### ISSUE TITLE
Android build error: appAuthRedirectScheme

### ISSUE THREAD

**Issue by bulats** [FormidableLabs/react-native-app-auth#139]
I'm getting the following build error: Error:Execution failed for task ':app:processDebugManifest'.
> Manifest merger failed : Attribute data@scheme at AndroidManifest.xml requires a placeholder substitution but no value for <appAuthRedirectScheme> is provided.

I've replaced appAuthRedirectScheme in build.gradle.

AndroidManifest.xml also looks weird: 

```
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
          package="com.reactlibrary">

</manifest>
```

The issue persists even if I remove: 

```
manifestPlaceholders = [
      appAuthRedirectScheme: 'io.identityserver.demo'
    ]
```

**Comment by kadikraman** [comment]
Are you sure it's an issue with this library if it persist even after removing the `manifestPlaceholders`?

I can only suggest checking out out [example app](https://github.com/FormidableLabs/react-native-app-auth/blob/6a3dfb0e004a1a4d658628f32e3eb7ec441d105c/Example/android/app/build.gradle#L100) and comparing your configuration.

**Comment by bulats** [comment]
@kadikraman Yes, I've tried removing it and adding it again as well.  Anyhow, thanks for the idea. Fingers crossed I'll get it working.

**Comment by djleonskennedy** [comment]
@kadikraman Thank you
